src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala

```diff
@@ -162,22 +162,22 @@ class OpenJdkMigrations {
       }
 
   @ChangeSet(
```

Comment:
please move the whole changeset to the end. next to changeset 078

Reply:
moved it to the end of the file.